packages/trimerge-sync/src/auto-merge-heads.ts

```diff
@@ -0,0 +1,45 @@
+import { AutoMergeFn, CommitDoc, DocAndMetadata } from './differ';
```

Comment:
what is the rule re: file naming? should we be using camel case or kebab case for files? or is there a rule about when to use one or the other?

Reply:
I've been using CamelCase for classes, and kebab-case for everything else, but this file is just poorly named

Reply:
my preference would be kebab-case for everything (that's the javascript norm), but it feels weird for classes